changelog/64909.deprecated.md

```diff
@@ -0,0 +1 @@
+Deprecate all the Apache modules
```

Comment:
Can we include the URL in the deprecation notice and define which release it will be removed.
```suggestion
Deprecate all the Apache modules in favor of the Apache Salt Extension https://github.com/salt-extensions/saltext-apache. The Apache modules will be removed in Salt core in 3009.0.
```